daq/faucet_states_collector.py

```diff
@@ -50,20 +112,26 @@ def process_port_state(self, dpid, port, status, timestamp):
             port_table.setdefault(
                 FaucetStatesCollector.MAP_ENTRY_PORT_STATE_CHANGED_COUNT, 0) + 1
 
-    @save_states
-    def process_port_learn(self, dpid, port, mac, timestamp):
+    @dump_states
+    def process_port_learn(self, dpid, port, mac, src_ip, timestamp):
```

Comment:
NB this will cause a merge conflict with Anurag's code -- whoever gets their first wins!